python/runtime/utils/PyRestRemoteClient.cpp

```diff
@@ -409,12 +409,14 @@ class PyNvcfRuntimeClient : public PyRestRemoteClient {
       if (m_logLevel > LogLevel::None) {
         auto queueDepth = getQueueDepth(m_functionId, m_functionVersionId);
         if (queueDepth.has_value() && queueDepth.value() > 0) {
-          cudaq::log("Number of jobs ahead of yours in the NVQC queue: {}.",
-                     queueDepth.value());
+          cudaq::log(
+              "Number of jobs ahead of yours in the NVQC queue: {}. Your job "
+              "will start executing once it gets to the head of the queue.",
+              queueDepth.value());
         }
       }
 
-      if (m_logLevel > LogLevel::None)
+      if (m_logLevel > LogLevel::Info)
         cudaq::log("Posting NVQC request now");
```

Comment:
We're intentionally make the "Posting NVQC request now" message not appear by default.